packages/jest-matchers/src/global_object.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Maybe we can call it `jestMatchersObject`? (btw would be nice to settle on naming convention for files)